Model/User.php

```diff
@@ -190,9 +176,7 @@ public function unserialize($serialized)
             $this->salt,
             $this->usernameCanonical,
             $this->username,
-            $this->expired,
             $this->locked,
-            $this->credentialsExpired,
```

Comment:
the issue is that the user is not also serialized in the session (which may not be cleared between deployments). I have an idea to keep compat with unserializing the old data. I will work on this when I have time for it.